Client/Reports/index.ts

```diff
@@ -82,7 +82,7 @@ export class Reports {
 		if (page)
 			params.push("page=" + page)
 		if (pageSize)
-			params.push("pageSize=" + pageSize)
+			params.push("size=" + pageSize)
 		if (includeCount)
 			params.push("includeCount=" + includeCount)
```

Comment:
i think you have to check if includeCount is not null (or undefined or whatever)

because some things might have the count as default true? so you could never set it false